sdk/src/logs/batch_log_processor.cc

```diff
@@ -292,13 +315,6 @@ void BatchLogProcessor::NotifyCompletion(
     synchronization_data->is_force_flush_notified.store(true, std::memory_order_release);
     synchronization_data->force_flush_cv.notify_one();
   }
-
```

Comment:
We need to call `async_export_waker.notify_all()` here, or all threads call Export and Shutdown will be blocked util timeout.

Reply:
Yeah will do this.